### PR TITLE
fix(Trader): Post wargames changes

### PR DESCRIPTION
**Pull request by chrismaree**
**Motivation**

After testing out the range trader on kovan, there were a few small changes that needed to be added. In particular:
- Uniswap updated on of their events when broke the v3 price feed
- inverted price feed support is required when making trades.

**Summary**

Fixes bots after wargames on kovan

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [X]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

**Issue(s)**

Fixes #XXXX

### CODE REVIEW

File: packages/trader/src/RangeTrader.ts [hunk 1]
```diff
@@ -165,6 +169,13 @@ export class RangeTrader {
       .mul(this.fixedPointAdjustment) // Scale the numerator before division
       .div(this.normalizePriceFeedDecimals(expectedValue));
   }
+
+  // TODO: there are a number of places through the repo that re-use this method. it should be refactored to a common util.
+  _invertPriceSafely(price: BigNumber) {
```
Comment (nicholaspai) [inline packages/trader/src/RangeTrader.ts:174]: +1 to this comment, maybe add it as a common util in this PR and we can refactor it elsewhere as we go?

File: packages/trader/src/RangeTrader.ts [hunk 2]
```diff
@@ -125,6 +125,10 @@ export class RangeTrader {
       desiredPrice: this.formatDecimalString(this.normalizePriceFeedDecimals(desiredPrice))
     });
 
+    if (this.tokenPriceFeed.invertPrice === "true") {
```
Comment (nicholaspai) [inline packages/trader/src/RangeTrader.ts:128]: why not just `if (this.tokenPriceFeed.invertPrice)`

**Review by nicholaspai — APPROVED** [review]
Left some nit comments, LGTM

File: packages/trader/src/RangeTrader.ts (hunk 1 above)
Comment (chrismaree) [reply packages/trader/src/RangeTrader.ts:174]: yup! there are a few places of refactoring I want to do like this. I think we have a task tracking it for next sprint

File: packages/trader/src/RangeTrader.ts (hunk 2 above)
Comment (chrismaree) [reply packages/trader/src/RangeTrader.ts:128]: it's a string not a bool!

**Pull request merged**